### PR TITLE
modify doc about deploy of hadoop: deprecate KERB_TICKET_CACHE_PATH

### PR DESCRIPTION
**Pull request by weberxie**
I have create https://github.com/tensorflow/tensorflow/issues/12290 to declare this problem.

### CODE REVIEW

**Comment by weberxie** [comment]
I signed it!

**Comment by weberxie** [comment]
I signed it!

File: tensorflow/docs_src/deploy/hadoop.md [hunk 1]
```diff
@@ -52,10 +52,16 @@ be set:
...
+If the Hadoop cluster is in secure mode, one the following environments variable
+should be set:
+*   **KRB5CCNAME**: The build in Kerberos environment variable for ticket cache file.
+For example:
 
-*   **KERB_TICKET_CACHE_PATH**: The path of Kerberos ticket cache file. For example:
+    ```shell
+    export KRB5CCNAME=/tmp/krb5cc_10002
+    ```
+*   **KERB_TICKET_CACHE_PATH**: The path of Kerberos ticket cache file, duplicate
```
Comment (jhseu) [inline tensorflow/docs_src/deploy/hadoop.md]: Can we just delete the mention of the old environment variable?

Can you also add a TODO comment next to KERB_TICKET_CACHE_PATH in the code to mention that it should be deleted later?

Comment (weberxie) [inline tensorflow/docs_src/deploy/hadoop.md]: > Can we just delete the mention of the old environment variable?
> 
> Can you also add a TODO comment next to KERB_TICKET_CACHE_PATH in the code to mention that it should be deleted later?

OK, thanks for your reply, according to your advice,  I'll retry to update this commit.

Comment (weberxie) [inline tensorflow/docs_src/deploy/hadoop.md]: @jhseu By the way, what's the proper time to delete KERB_TICKET_CACHE_PATH?

Comment (jhseu) [inline tensorflow/docs_src/deploy/hadoop.md]: We'll delete it as part of TensorFlow 2.0. Added it to our tracker internally.

**Review by jhseu — APPROVED** [review]

**Pull request merged**